File: Java/adocaoanimais/src/main/java/br/com/iteris/universidade/adocaoanimais/domain/entity/Animal.java
package br.com.iteris.universidade.adocaoanimais.domain.entity;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDate;
import java.util.Date;

@Data
@AllArgsConstructor
public class Animal {
    private int id;
    private String nome;
    private int idade;
    private String especie;
    private LocalDate dataNasimcento;
    private int fofura;
    private int carinho;
    private String email;

}

File: C#/AdotaAnimais/AdotaAnimais/Domain/Entity/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AdotaAnimais.Domain.Entity
{
    public class Animal
    {

        public int Id { get; set; }
        public string Nome { get; set; }
        public int Idade { get; set; }
        public string Especie { get; set; }
        public DateTime? DataNascimento { get; set; }
        public int Fofura { get; set; }
        public int Carinho { get; set; }
        public string Email { get; set; }
    }
}
